modules/salt/pages/yomi.adoc

```diff
@@ -36,47 +53,63 @@ zypper in yomi-formula
 systemctl restart salt-master.service
 ----
 
-This package contains the Salt states and modules that describe the Yomi state, the {productname} formulas that will allow the creation of the pillar via the web interface, the documentation about the different sections of the pillar, and some examples about how to parameterize installations based on openSUSE, MicroOS or SLE.
 
-To find the new modules, this package is already adding a new configuration file in  [path]``/etc/salt/master.d``, named yomi-formula.conf.
+// For more information about the Yomi formula, see xref:salt:formula-yomi.adoc[] TBC --LKB 2020-03-03
```

Comment:
Is this a doc that we need to provide too?

Reply:
We will need to get the formula details into the Salt Guide with the other formulas. I'll create an issue so we don't forget about it.

Reply:
https://github.com/uyuni-project/uyuni-docs/pull/137#discussion_r391973906